supertokens_python/recipe/emailpassword/types.py

```diff
@@ -84,3 +87,10 @@ def __init__(
     TypeEmailPasswordPasswordResetEmailDeliveryInput,
```

Comment:
Similar to `TypeEmailVerificationEmailDeliveryInput`, you don't require user_context in `TypeEmailPasswordPasswordResetEmailDeliveryInput`